sql/moz-fx-data-shared-prod/adjust/adjust_kpi_deliverables/view.sql

```diff
@@ -2,7 +2,7 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.adjust.adjust_kpi_deliverables`
 AS
 SELECT
-    date as submission_date,
+    date,
```

Comment:
why was this changed to a more generic column name?

Reply:
Mostly semantics. This date is when the data was aggregated rather than when it was submitted.